WooCommerce/Classes/ViewRelated/Orders/Order Filters/FilterOrderListViewModel.swift

```diff
@@ -281,20 +309,39 @@ extension FilterOrdersByProduct: FilterType {
 // MARK: - Constants
 private extension FilterOrderListViewModel {
     enum Localization {
-        static let filterActionTitle = NSLocalizedString("Show Orders", comment: "Button title for applying filters to a list of orders.")
+        static let filterActionTitle = NSLocalizedString(
+            "filterOrderListViewModel.OrderListFilter.filterActionTitle",
+            value: "Show Orders",
+            comment: "Button title for applying filters to a list of orders.")
     }
 }
 
 private extension FilterOrderListViewModel.OrderListFilter {
     enum Localization {
-        static let rowTitleOrderStatus = NSLocalizedString("Order Status", comment: "Row title for filtering orders by order status.")
-        static let rowTitleDateRange = NSLocalizedString("Date Range", comment: "Row title for filtering orders by date range.")
-        static let rowTitleProduct = NSLocalizedString("filterOrderListViewModel.OrderListFilter.rowTitleProduct",
-                                                       value: "Product",
-                                                       comment: "Row title for filtering orders by Product.")
-        static let rowCustomer = NSLocalizedString("filterOrderListViewModel.OrderListFilter.rowCustomer",
-                                                   value: "Customer",
-                                                   comment: "Row title for filtering orders by customer.")
+        static let rowTitleOrderStatus = NSLocalizedString(
+            "filterOrderListViewModel.OrderListFilter.rowTitleOrderStatus",
+            value: "Order Status",
+            comment: "Row title for filtering orders by order status.")
+
+        static let rowTitleDateRange = NSLocalizedString(
+            "filterOrderListViewModel.OrderListFilter.rowTitleDateRange",
+            value: "Date Range",
+            comment: "Row title for filtering orders by date range.")
```

Comment:
There's no real benefit to changing existing localized string definitions to use the `key, value, comment` initialiser, AFAIK. 

It incurrs cost/work, because the translation will need to be done again in all the langages even though it's not changed, and I don't know if Glotpress is good at linking up the old version with the new version...